src/main/java/com/notably/logic/parser/DeleteCommandParser.java

```diff
@@ -33,12 +33,14 @@ public DeleteCommandParser(Model notablyModel) {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_TITLE);
 
+        String title;
         if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE)
                 || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format("Invalid Command"));
+            title = args.trim();
```

Comment:
Let's check for an empty title. That is, throw a `ParseException` if the user types `delete` only, without any arguments!

Reply:
Done!